features/fixtures/ios-swift-cocoapods/iOSTestApp/AppDelegate.swift

```diff
@@ -13,19 +13,10 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         return true
     }
 
-    func triggerEvent(name: String, delay: TimeInterval) {
```

Comment:
Allowing the specification of a time delay was intentional to account for cases where exceptions happen after a specific duration, like a crash on launch vs triggering click events on a normal human time scale which lead to a crash.

Reply:
The method signature has changed but this behaviour now remains in place.